security/advancedtls/crl_provider_test.go

```diff
@@ -141,7 +141,6 @@ func (s) TestFileWatcherCRLProvider(t *testing.T) {
 	t.Parallel()
 	const nonCRLFilesUnderCRLDirectory = 5
 	nonCRLFilesSet := make(map[string]struct{})
-	//var callbackMutex sync.Mutex
 	var nonCRLMutex sync.Mutex
```

Comment:
So can you delete this now?

Reply:
Yeah, there is no need to protect this local to the test map anymore. We do the read once provider's goroutine is done